.changeset/afraid-pandas-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
+---
+
+Add debug command LLD
```

Comment:
Please remove the changeset, it is not worth bumping the version for this since it does not change anything in the LLD code.

Reply:
Done